docs.markdown

```diff
@@ -1291,416 +1399,580 @@ smooks.filterSource(new StreamSource(csvReader), result);
+</smooks-resource-list>
+```
 
-By default, segments, fields, and components are not truncatable.
+The DFDL cartridge supports variables, on disk caching, and trace debugging. Consult the [XSD documentation](/xsd/smooks/dfdl-1.0.xsd) for further
+information.
 
-So, a \<field>, \<component> and \<sub-component> can be present in a message in one of the following states:
 
-1.  Present with a value _(required="true")_
-2.  Present without a value _(required="false")_
-3.  Not Present _(required="false" and truncatable="true")_
+EDI Cartridge
+-------------
 
-### Imports
+### Maven Coordinates
 
-Many message groups use the same segment definitions. Being able to define these segments once and import them into a top level configuration saves on a lot of duplication. A simple configuration demonstrating the import feature would be as follows:
 ```xml
-<?xml version="1.0" encoding="UTF-8"?>  
-<medi:edimap xmlns:medi="https://www.smooks.org/schema/edi-message-mapping-1.7.xsd">  
-   
-    <medi:import truncatableSegments="true" truncatableFields="true" truncatableComponents="true" resource="example/edi-segment-definition.xml" namespace="def"/>  
-   
-    <medi:description name="DVD Order" version="1.0"/>  
-   
-    <medi:delimiters segment="&#10;" field="*" component="^" sub-component="~" escape="?"/>  
-   
-    <medi:segments xmltag="Order">  
-        <medi:segment minOccurs="0" maxOccurs="1" segref="def:HDR" segcode="HDR" xmltag="header"/>  
-        <medi:segment minOccurs="0" maxOccurs="1" segref="def:CUS" segcode="CUS" xmltag="customer-details"/>  
-        <medi:segment minOccurs="0" maxOccurs="-1" segref="def:ORD" segcode="ORD" xmltag="order-item"/>  
-    </medi:segments>  
-   
-</medi:edimap>
-```
-The configuration-example above demonstrates the use of import that were introduced in Smooks v1.1, where single segments or segments containing childsegments can be separated into a separate file for better reuse in the future.
+<dependency>
+    <groupId>org.smooks.cartridges.edi</groupId>
+    <artifactId>smooks-edi-cartridge</artifactId>
+    <version>2.0.0</version>
+</dependency>    
+``` 
 
-*   **segref**: Contains a "namespace:name" referencing the segment to import.
-*   **truncatableSegments**: Overrides the 'truncatableSegments' specified in the imported resource mapping file.
-*   **truncatableFields**: Overrides the 'truncatableFields' specified in the imported resource mapping file.
-*   **truncatableComponents**: Overrides the 'truncatableComponets' specified in the imported resource mapping file.
+### XML Namespaces
 
-### Type Support
+```
+xmlns:edi="https://www.smooks.org/xsd/smooks/edi-2.0.xsd"
+xmlns:dfdl="https://www.smooks.org/xsd/smooks/dfdl-1.0.xsd"
```

Comment:
Did you mean to have the `dfdl` namespace listed here too ?

If you did then maybe a line or two on why it's needed would be useful.

Reply:
Okay ... I see the same on the `edifact` cartridge so it must be intentional. I think a line explaining why they're all needed would be good.